### PR TITLE
RaggedTensor support for mean_squared_logarithmic_error.

### PR DESCRIPTION
**Pull request by pedro-r-marques**
Follow up to #46283.

### CODE REVIEW

File: tensorflow/python/keras/losses_test.py [hunk 1]
```diff
@@ -632,6 +632,15 @@ def test_zero_weighted(self):
...
     self.assertAlmostEqual(self.evaluate(loss), 0.0, 3)
 
+  def test_ragged_tensors(self):
+    msle_obj = losses.MeanSquaredLogarithmicError()
+    y_true = ragged_factory_ops.constant([[1, 9, 2], [-5, -2]])
+    y_pred = ragged_factory_ops.constant([[4, 8, 12], [8, 1]],
+                                         dtype=dtypes.float32)
+    sample_weight = constant_op.constant([1.2, 3.4], shape=(2, 1))
+    loss = msle_obj(y_true, y_pred, sample_weight=sample_weight)
+    self.assertAlmostEqual(self.evaluate(loss), 5.1121, 3)
```
Comment (tomerk) [inline tensorflow/python/keras/losses_test.py:642]: Can you add some comments to this test noting the actual computation you expect to take place here and why it should come out to 5.11? (e.g. how you did in one of the other losses

**Review by tomerk — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/losses_test.py (hunk 1 above)
Comment (pedro-r-marques) [reply tensorflow/python/keras/losses_test.py:642]: Done.

Comment (tomerk) [reply tensorflow/python/keras/losses_test.py:642]: You might need to push again?

Comment (pedro-r-marques) [reply tensorflow/python/keras/losses_test.py:645]: My bad. Sorry.

**Review by tomerk — APPROVED** [review]
Thanks so much again for putting all this together!

**Comment by rthadur** [comment]
@pedro-r-marques can you please check sanity build failures ?

**Comment by pedro-r-marques** [comment]
@rthadur Addressed pylint error. Would you please re-trigger the CI ? thanks.

**Review by rthadur — APPROVED** [review]

**Review by tomerk — APPROVED** [review]

**Pull request merged**